prow/plugins/trigger/pull-request.go

```diff
@@ -280,7 +276,7 @@ I understand the commands that are listed [here](https://go.k8s.io/bot-commands?
 	} else {
 		comment = fmt.Sprintf(`Hi @%s. Thanks for your PR.
 
-I'm waiting for a [%s](https://github.com/orgs/%s/people) %smember to verify that this patch is reasonable to test. If it is, they should reply with `+"`/ok-to-test`"+` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](%s) to skip this step.
+I'm waiting for a [%s] %smember to verify that this patch is reasonable to test. If it is, they should reply with `+"`/ok-to-test`"+` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](%s) to skip this step.
```

Comment:
Not sure why we here remove this link

Reply:
there are 6 args required and we had 7, removed the corresponding trustedOrg arg which was part of link

Reply:
yeah but you alter md and there will be no link if you do sth like this